Analysis/law/setup.sh

```diff
@@ -1,17 +1,22 @@
 #!/usr/bin/env bash
 
 action() {
-    local this_file="$( [ ! -z "$ZSH_VERSION" ] && echo "${(%):-%x}" || echo "${BASH_SOURCE[0]}" )"
-    local this_dir="$( cd "$( dirname "$this_file" )" && pwd )"
+    eosfusebind -g
```

Comment:
Do we need to run `eosfusebind` also here? If yes, do the same check as in `bootstrap.sh`.